novawallet/Common/View/RoundedGradientBackgroundView.swift

```diff
@@ -40,6 +45,15 @@ final class RoundedGradientBackgroundView: RoundedView {
         rightGradientView.endPoint = model.right.endPoint
     }
 
+    func bind(model: GradientModel) {
```

Comment:
The first one utilizes both right and left gradients and the second is used for single gradient layer approach.
The other option was to make both fields of banner model optional, but it would require to make changes everywhere it's used.